lib/fa-harness-tools/github_client.rb

```diff
@@ -52,6 +63,28 @@ def get_commit_sha(short_sha)
       commit[:sha]
     end
 
+    # Return a full commit SHA from a tag
+    #
+    # The `tag` argument should be a Hash of tag data with an :object that can
+    # either be an annotated tag or a commit object.
+    #
+    # @return [String] Full commit SHA
+    # @raise [LookupError] If tag cannot be found
+    def get_commit_sha_from_tag(tag)
```

Comment:
Is it worth adding tests to cover this behaviour?

Reply:
Yeah, maybe I can introduce some with a bit of stubbing, will have a look! I would like to add full on integration tests (#3) but today's probably not that day 🙂

Reply:
If you want to merge this and double back to the tests at a later time, feel free. It might be more important to get it out than stubbing and mocking the client?

Reply:
That's OK, was just finishing the first round of specs for this method 👍